singularity-eos/eos/eos_eospac.hpp

```diff
@@ -53,10 +54,91 @@ using namespace eos_base;
 #define SG_PIF_NOWARN
 #endif // !defind SG_PIF_NOWARN
 
+namespace impl_eospac {
+
+inline void SetUpDensityTemperatureScalingOptions(EOS_INTEGER options[],
+                                                  EOS_REAL values[], EOS_INTEGER &nopts,
+                                                  Transform &transform) {
```

Comment:
This and others can be `const Transform &`

Reply:
Done